mappings/markdown/Zeek/v1.3.0/ssh_log/README.md

```diff
@@ -0,0 +1,78 @@
+| `count`                        | `auth_attempts`             | The number of authentication attempts observed.                                         | Integer                 |
+| `protocol_ver`                 | `version`                   | SSH major version (1, 2, or unset).                                                     | As string for vendor compatibility. |
+
+
+ ### Conditional mapping:
+Fields described here are subject to dynamic mappings contingent on a conditional evaluation of source data.
+| OCSF                           | Raw               | Zeek Field Description                                              | Evaluation Conditions                                                                   |
+| ------------------------------ | ----------------- | ------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| `activity_id`                  | `auth_success`    | Authentication result.                                              | If "true" then "99", <br>if "false" then "4", <br>else "6". <br>Type is Integer.        |
+| `activity_name`                | `auth_success`    | Authentication result.                                              | If "true" then "Authorization Successful", <br>else defined by activity_id.             |
+| `type_uid`                     | `auth_success`    | Authentication result.                                              | If "true" then "400799", <br>if "false" then "400704", <br>else "400706". <br>Type is Integer. |
+| `connection_info.direction_id` | `direction`       | Direction of the connection. If the client was a local host logging into an external host, this would be OUTBOUND. INBOUND would be set for the opposite situation. | If "INBOUND" then "1", <br>if "OUTBOUND" then "2", <br>else "3". |
+| `client_hassh.fingerprint.algorithm_id` | `hassh`  | The hassh information.                                              | If `hassh` is present, then "1". (MD5) <br>Type is Integer.                             |
+| `server_hassh.fingerprint.algorithm_id` | `hasshServer` | The hasshServer information.                                   | If `hasshServer` is present, then "1". (MD5) <br>Type is Integer.                       |
+
+ ### Unmapped:
+| OCSF       | Raw               | Zeek Field Description                                              |
+| ---------- | ----------------- | ------------------------------------------------------------------- |
+| `unmapped` | `hasshVersion`    | The hasshVersion information.                                       |
+| `unmapped` | `hasshAlgorithms` | The hasshAlgorithms information.                                    |
+| `unmapped` | `hasshServerAlgorithms` | The hasshServerAlgorithms information.                        |
```

Comment:
Is `haasshAlgorithms/hasshServerAlgorithms ` a list of algorithms? Or could we use `client_hassh.fingerprint.algorithm` / `server_hassh.fingerprint.algorithm`?

Reply:
From the [OCSF HASSH](https://schema.ocsf.io/1.3.0/objects/hassh) page:

> algorithm | recommended | String | The concatenation of key exchange, encryption, authentication and compression algorithms (separated by ';').
> -- | -- | -- | --